docs/tutorials/tut-iiot-cost-estimation.md

```diff
@@ -0,0 +1,134 @@
+
+[Home](readme.md)
+
+## Overview
+The costs of running the [Azure IIoT Platform](https://github.com/Azure/Industrial-IoT) in Azure depend on factors such as:
+* Number of OPC UA servers (e.g. PLCs)
+* Number of nodes per server
+* Frequency of data changes
+* Node data types
+
+In order to estimate your monthly costs, follow the steps below to create a simulation to gather the data for the calculation.
+
+### IoT Hub
+While there are several resources needed to run the IIoT Cloud Platform, most of the costs are incurred by the IoT Hub. The [pricing](https://azure.microsoft.com/en-us/pricing/details/iot-hub/) page lists a few editions with different daily message limits:
+
+| IoT Hub size | Max. msg/day  |
+| ------------ | ------------: |
+| S1	         |       400,000 |
+| S2           |     6,000,000 |
+| S3           |   300,000,000 |
+
+### IoT Hub pricing
+ The pricing page also mentions *MESSAGE METER SIZE*, which is the measure of the message chunks that are discounted from the daily contingent. If a device-to-cloud (D2C) message is within 4 KB, it will be charged as 1 message. However, if the message exceeds 4 KB by even 1 byte, it will be charged as 2 messages. See [Azure IoT Hub pricing information](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-pricing) and [Message size](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-messages-construct#message-size) for more details.
+
+### IoT Hub message optimization
+Crafting message chunks in an optimal way can lead to large savings. For instance, consider 10 IoT devices that send a 1-KB message every second. These messages will be charged as 10 message chunks per second in total. Consequently, you will need IoT Hub capacity for 10 * 86,400 = 864,000 messages per day, hence at least three S1 units. 
+
+However, if every device instead sends a 4-KB message every 4 seconds, the messages will be charged as 10 message chunks every 4 seconds. This translates to 86,400 / 4 = 21,600 messages per device, for a total of 216,000 messages for all 10 devices per day. In this case, one S1 unit would be enough.
+
+If the requirements state that waiting 4 seconds for the data is not acceptable, that will of course affect the monthly costs. See [IoT Hub quotas and throttling](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-quotas-throttling) and [IoT Hub limits](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#iot-hub-limits) for further relevant information.
+
+### Reference factories
+The following factory setups are provided for reference. A slow node changes every 10 s, a fast one every 1 s.
+
+| Factory size | PLCs | Slow nodes | Fast nodes | Values/s/PLC | VM size | IoT Hub |
+| ------------ | ---: | ---------: | ---------: | -----------: | ------- | ------- |
+| Small	       |   25 |        250 |         50 |           75 | >= B2ms | 8 * S1  |
```

Comment:
is here a tab/space too much?

Reply:
No, this renders fine in VS Code. Must be a GH issue.
![image](https://user-images.githubusercontent.com/1353540/87546175-64bb7180-c6a9-11ea-843e-0b1709f81a85.png)